## 6. Enrichment and Depletion {.page_break_before}
In order to study low abundance protein modifications, or to study rare proteins in complex mixtures, various methods have been developed to enrich or deplete specific proteins or peptides. 

### Peptide enrichment 

#### Phosphoproteomics  
Protein phosphorylation, a hallmark of protein regulation, dictates protein interactions, signaling, and cellular viability. 
This post-translational modification (PTM) involves the installation of a negatively charged phosphate moiety (PO 4-) onto the hydroxyl side-chain of serine or threonine residues on target proteins. 
Additionally, while less commonly modified than serine and threonine, histidine [@DOI:10.1038/nmeth.4580; @DOI:10.1021/ac0707838; @DOI:10.1038/s41592-022-01524-0], arginine [@DOI:10.1002/pmic.201200240], and tyrosine [@DOI:10.1073/pnas.2436191100; @DOI:10.1158/0008-5472.can-20-3804; @DOI:10.1016/j.cell.2006.09.026] phosphorylation also represent important cell signaling biology. 
Protein kinases catalyze the transfer of PO 4- group from ATP to the nucleophile (OH) group of serine, threonine, and tyrosine residues, while protein phosphatases catalyze the removal of PO4-. 
Phosphorylation changes the charge of a protein, often altering protein conformation and therefore function [@PMID:26473910]. 
Protein phosphorylation is one of the major PTMs that alters the stability, subcellular location, enzymatic activity complex formation, degradation of protein, and cell signaling of protein with a diverse role in cells [@PMID:31819260, @PMID:35227377; @PMID:32676006]. 
Phosphorylation can regulate almost all cellular processes, including metabolism, growth, division, differentiation, apoptosis, and signal transduction pathways [@PMID:28656226]. 
Rapid changes in protein phosphorylation are associated with several diseases [@PMID:11589691]. 

Several methods are used to characterize phosphorylation using modification-specific enrichment techniques combined with advanced MS/MS methods and computational data analysis [@PMID:19743430]. 
There are many challenges with studying phosphorylation [@PMID:24890697]. 
For example, many phosphopeptides are low stoichiometry compared to non-phosphorylated peptides, which makes them difficult to identify. 
Phosphopeptides also exhibit low ionization efficiency [@PMID:28283928]. 
To overcome these challenges, it is important to reduce sample complexity to detect large numbers of phosphorylation sites. 
This is accomplished using enrichment the modified proteins and/or peptides [@PMID:19816929; @PMID:30457327; @PMID:22002794].

As with any proteomics experiment, phosphoproteomics studies require protein extraction, proteolytic enzyme digestion, phosphopeptide enrichment, peptide fractionation, LC-MS/MS, bioinformatics data analysis, and biological function inference. 
Special consideration is required during protein extraction where the cell lysis buffer should include phosphatase inhibitors such as sodium orthovanadate, sodium pyrophosphatase and beta-glycerophosphate [@PMID:35011700]. 

Enrichment can be done at the protein level before proteolysis. 
Phosphoprotein enrichment typically involves the use of immobilized metal-affinity chromatography (IMAC) to selectively capture phosphorylated proteins based on their high-affinity binding to metal ions such as Ga(III), Fe(III), Zn(II) and Al(III) [@PMID:10424175; @PMID:16464107; @PMID:3085541; @PMID:17480206; @PMID:16252304]. 

Enrichment is more commonly performed at the peptide level because there are several advantages over phosphoprotein-level enrichment. 
First, peptides have simpler three-dimensional structures than proteins, which makes them easier to separate and analyze. 
Second, phosphopeptide enrichment is not hindered by small, lipophilic, and very acidic or alkaline proteins [@PMID:22002794].
Third, prefractionation techniques such as strong anion exchange chromatography (SAX), strong cation exchange chromatography (SCX) and hydrophilic interaction chromatography (HILIC) are easier to use for peptide separation than they are for protein separation, and they are more sensitive than 2D-gel electrophoresis that is often used for intact proteins [@PMID:15966007; @DOI:10.1021/pr500893m]. 
As a result, phosphopeptide enrichment has yielded more experimental data than phosphoprotein enrichment [@PMID:17480206]. 
Phosphopeptide enrichment is typically done after any isobaric labeling strategy, although several have investigated the importance of order at these stages.

Phosphopeptide enrichment often uses titanium dioxide (TiO2) [@PMID:15858219] and/or IMAC such as Fe3+ coupled to solid-phase materials [@PMID:3085541; @PMID:35011700; @PMID:26584922]. 
The most common cost-effective beads for phosphopeptide extraction with Ti are ReSyn and GL Sciences, and CubeBio for Fe-based beads.
Often organic acids such as glutamic acid, lactic acid, glycolic acid is added to compete with acidic non-phosphopeptides for binding to the metal-ions.
Carr and coworkers even demonstrated phosphoproteome analysis without any enrichment [@DOI:10.1006/abio.1996.0313].

The use of Fe-IMAC column chromatography allows for the improved phosphopeptide from complex peptide mixtures [@PMID:25394399]. 
Compared to other formats like StageTips or batch incubations with TiO2 or Ti-IMAC beads, Fe-IMAC columns do not suffer from problems with poor binding or elution of phosphopeptides, and the efficiency of enrichment increases linearly with the amount of starting material [@PMID:28188522].
Also with recent improvements to Ti based beads, the MagReSyn Ti-IMAC HP with Ti4+ attached with a flexible linker (to reduce steric hindrance) activated with phosphonate groups for Ti4+ chelation, and the MagReSyn Zr-IMAC HP, also with a flexible linker activated with phosphonate groups for Zr4+ chelation, have shown superior phosphopeptide extraction as compared to FE-IMAC.

Multiple IMAC steps can be used in parallel or sequentially to improve phosphopeptide coverage. 
Lai et al., (2012) showed that the combined use of Fe3+-IMAC and Ti(4+)-IMAC chromatography enables complementary identification of more phosphorylation sites than either technique alone [@PMID:22886815]. 
A novel phosphopeptide enrichment technique using sequential enrichment with magnetic Fe3O4 and TiO2 particles was developed to detect mono- and multi-phosphorylated peptides [@PMID:20411586]. 

More recently, the use of  Src Homology 2 (SH2) domains as specific affinity reagents for phosphotyrosine is an emerging technology allowing an expanded fractionation of tyrosine phosphopeptides. Here, short chain protein domains have been constructed and affinity enhanced through yeast two hybrid screening to arrive at high affinity matrices capable of outperforming traditional IMAC approaches [@PMID:27880036; @PMID:37668982].

#### Tips for studying phosphorylation:  
•	Cell lysates should always be prepared using phosphatase inhibitors and samples should be placed on the ice during sonication for protein extraction. 
•	Increase the amount of starting material of your sample for phosphoenrichment to at least 1 mg of protein or more for optimal results.
•	If using anti-phosphorylation antibodies, ensure their specificity is confirmed with other methods.
•	Make sure to select a suitable method for the phosphoenrichment that fits the experiment goals.
•	TiO2-based phosphopeptide enrichment methods have different enrichment specificities; selecting non-phosphopeptide excluders such as glutamic acid, lactic acid, glycolic acid, and dihydroxybenzoic acid are the key part of the study [@PMID:35805136].
•	Do not use milk as a blocking agent when western blotting for phosphorylation because milk contains the phosphoprotein casein and can lead to a higher background due to non-specific binding.

#### Glycosylation
Mass spectrometry-based analysis of protein glycosylation has emerged as the premier technology to characterize such a universal and diverse class of biomolecules.
Glycosylation is a heterogenous post-translational modification that decorates many proteins within the proteome, conferring broad changes in protein activity. [@DOI:10.1093/glycob/12.4.43R; @DOI:10.1016/j.cell.2006.08.019] 
This PTM can take many forms.
The covalent linkage of mono- or oligosaccharides to polypeptide backbones through a nitrogen atom of asparagine (N) or an oxygen atom of serine (S) or threonine (T) side-chains creates N- and O-glycans, respectively.
The heterogenity of proteoglycans is not directly tied to the genome, and thus cannot be inferred. 
Rather, the abundance and activity of protein glycosylation is governed by glycosyltransferases and glycosidases which add and remove glycans, respectively. 
The fields of glycobiology and bioanalytical chemistry are intricately intertwined with mass spectrometry at the center thanks in part to its power of detecting any modification that imparts a mass shift. 

Due to the myriad glycan structures and proteins which harbor them, the enrichment of glycoproteins or glycopeptides is not as streamlined as that of other PTMs [@DOI:10.1074/mcp.R120.002277]. 
The enrichment of glycoproteome from the greater proteome inherently introduces bias prior to the LC-MS/MS analysis.
One must take into account which class or classes of glycopeptides they are interested in analyzing before enrichment for optimal LC-MS/MS results. 
Glycopeptides can be enriched via glycan affinity, for example to glycan-binding proteins, chemical properties like charge or hydrophilicity, chemical coupling of glycans to stationary phases, and by bioorthogonal, chemical biology approaches.
Glycan affinity-based enrichment strategies include the use of lectins, antibodies, inactivated enzymes, immobilized metal affinity chromatography (IMAC), and metal oxide affinity chromatography (MOAC).
The enrichment of glycopeptides by their chemical properties, for example by biopolymer charge and hydrophobicity, include hydrophilic interaction chromatography (HILIC), electrostatic repulsion-hydrophilic interaction chromatography (ERLIC), and porous graphitic carbon (PGC).
One variation of ERLIC combines strong anion exchange, electrostatic repulsion, and hydrophilic interaction chromatography (SAX-ERLIC) has risen in popularity thanks to robustness and commercially available enrichment kits [@DOI:10.1007/978-1-0716-1241-5_8; @DOI:10.1021/acs.jproteome.6b00849].

Chemical coupling methods most often used to enrich the glycoproteome employ hydrazide chemistry for sialylated glycopeptides.
Glycan are cleaved from the stationary phase by PNGase F.
The dependence of chemical coupling methods on PNGase F biases their output toward N-glycopeptides.
Alkoxyamine compounds and boronic acid-based methods have also shown utility. 
We direct readers to several reviews on glycopeptide enrichment strategies [@DOI:10.1074/mcp.R120.002277; @DOI:10.1016/j.ab.2009.08.023; @DOI:10.1016/j.bbapap.2014.05.002; @DOI:10.1039/C3RA42960A; @DOI:10.1021/acs.analchem.6b04343]

#### Antibody enrichments of modifications  
Western blot analysis is used to detect the PTMs in a protein through the use of antibodies [@PMID:25059473]. 
As an extension, pan-PTM antibodies have been used to isolate peptides bearing the PTM of interest [@PMID:8633009]. 
One benefit of this approach is that peptides are less likely to experience non-specific binding than proteins [@PMID:19743430]. 
Initially peptide immunoaffinity precipitation was developed to enrich for phosphotyrosine-containing peptides. 
This protocol was initially designed to enrich for phosphotyrosine-containing peptides [@PMID:15592455]. 
Peptide immunoprecipitation yielded significantly greater coverage of the phosphotyrosine proteome than global phosphorylation enrichment strategies by enriching for a subset of the phosphoproteome.  
Since then, peptide immunoaffinity precipitation has been used successfully to enrich for peptides with other phosphorylation motifs [@PMID:17525332; @PMID:20736484] as well as peptides with other modifications such as the diglycyl-lysine residue of ubiquitin modification after trypsin proteolysis [@PMID:21963094; @PMID:21906983; @PMID:21890473], acetyl-lysine [@PMID:19608861; @PMID:23201123; @PMID:25953088; @PMID:31087289; @DOI:10.1002/pmic.201800123], arginine methylation [@PMID:24129315], tyrosine nitration [@PMID:28714690], and tyrosine phosphorylation [@PMID:19770167; @PMID:15951569]. 

The O-linked β-D-N-acetylglucosamine (O-GlcNac) is found on serine and threonine residues and is involved in involved in the progression of cancers in multiple systems throughout the body [@PMID:36104770]. 
Anti-O-GlcNAc monoclonal antibody enables enrichment from O-GlcNAcylated peptides of cells and tissues. 
These antibodies have high sensitivity and specificity toward O-GlcNAc-modified peptides and do not identify O-GalNAc or GlcNAc in extended glycans [@PMID:34678516].

### Protein depletion (Blood samples)
Many plasma proteomics studies involve the analysis of plasma [@DOI:10.1021/acs.jproteome.7b00623; @DOI:10.1016/j.cels.2020.10.003].
However, the abundance range of proteins in the blood/plasma proteome exceeds 10 orders of magnitude. 
Due to this wide dynamic range, detection of proteins with medium and low abundance by proteomic analyses is difficult [@PMID:20677825], and identifying protein biomarkers from biological samples such as blood is often obstructed by proteins present at higher concentrations. 
In fact, the top 14 most abundant proteins in human plasma constitute over 99% of the total protein mass.
The removal of these high-abundant proteins enables the detection of less abundant proteins. 
The ability to deplete abundant proteins with specificity, reproducibility, and selectivity is extremely important in proteomic studies [@PMID:16052628].

The following are some of the methods used for abundant protein depletion:  

#### Dye-ligand depletion: 
This method is used for the depletion of serum albumin based on the interaction between albumin and dyes like Cibacron Blue (CB) through electrostatic force, hydrogen bonding and hydrophobic interactions. 
The method is relatively low cost, widely available, robust and has high binding capacity. 
However, it lacks specificity and has varying efficiency [@PMID:11694290; @PMID:24168355].

#### Protein-ligand depletion: 
This method is used for depletion of immunoglobulins (Ig) based on the interaction between the Fragment crystallizable (Fc) region of these Igs [@PMID:2473373] and cell wall protein A, G or A/G of Staphylococcus aureus and Streptococcus spp [@PMID:2938951; @PMID:10805799]. 
It is highly selective and has high yield and purity. 
However, non-specific binding may occur due to co-absorption of other proteins [@PMID:31617391].

#### Immunodepletion: 
This method is used for depletion of proteins having high abundance in plasma or serum on the basis of the specific interaction of these proteins with their respective antibodies (antigen-antibody interaction) [@PMID:27896769]. 
Immunodepletion has high specificity and commercial kits based on columns (Agilent MARS column) or loose beads (Thermo-Fisher High-Select depletions beads) both deplete the Top 14 most abundant protein in blood and are also readily available but expensive.
For some protocols, non-specific binding to these immunodepletion columns or beads by proteins of interest is highly dependent on washing conditions [@PMID:31617391].

#### Combinatorial peptide ligand library: 
This method is used for partial depletion of major proteins i.e., those with high abundance and for relative enrichment of lower and medium abundant proteins [@PMID:18451796]. 
It is based on the interaction with an array of ligands which are essentially peptides of 6 amino acids in length. 
It is also used for normalization of the global protein abundance [@PMID:25384740]. 
However, the drawbacks include non-specific binding as well as loss of proteins due to incomplete elution or inefficient binding [@PMID:31617391].

#### Precipitation: 
This method of abundant protein depletion works by altering the solubility of proteins using a chemical reagent including inorganic salt solution [@PMID:21963274], organic solvents [@PMID:25083595], non-ionic polymer [@PMID:27832179] and reducing agents [@PMID:19454248]. 
It is extremely simple and cost-effective. 
However, it is less specific with a risk of protein loss, difficulty in protein resolubilization as well as time consuming [@PMID:31617391].

#### New technologies: 
Newer methods of highly abundant protein depletion are based on the interaction between polymers such as bacterial cellulose nanofibers [@PMID:30219335], cryogels [@PMID:23668981], and nanomaterials [@DOI:10.1016/j.procbio.2010.07.007].  
These techniques are highly specific, relatively cheap, and very stable. 
They can also be reused since they have larger binding capacity and less cross-reactivity [@PMID:31617391].

Protein enrichment/depletion strategies which make use of protein coronas [@DOI:10.1038/s41467-020-17033-7; @DOI:10.1101/2023.08.28.555225] or extracellular vesicle enrichment [@DOI:10.1101/2023.06.10.544439] are enabling researchers to probe deeper into the plasma, serum, lymph, and cerebrospinal fluid proteomes.
Automated nanoparticle (NP) protein corona-based proteomics workflows are a novel approach to perform deep blood-based proteomics analysis at unprecedented protein IDs above 6,000 proteins [@PMID:37693476]. 
NPs can efficiently compress the dynamic range of protein abundances into a mass spectrometry accessible detection range and allow full automation of the protein preparation process providing a platform that can rival affinity based approaches with equivalent reproducibility and sensitivity [@PMID:32699280]. 
